content/en/docs/usage/kubectl-plugin.md

```diff
@@ -79,4 +81,22 @@ The default output will be printed to stdout in YAML format. One can use -o opti
 For example this will output `cert.yaml` in the latest API version:
 ```console
 kubectl cert-manager convert -f cert.yaml
+```
+
+### Create
+`kubectl cert-manager create` can be used to create cert-manager resources manually. Sub-commands are available
+to create different resources:
+#### certificaterequest
+To create a cert-manager CertificateRequest, use `kubectl cert-manager create certificaterequest`. The command takes in the name of the CertificateRequest to be created, 
+and creates a new CertificateRequest resource based on the YAML description of a Certificate resource as specified by `--from-certificate-file` flag, by generating a private key locally and creating a 'certificate signing request' 
+to be submitted to a cert-manager Issuer. The private key will be written to a local file, where the default is `<name_of_cr>.key`, or it can be specified using the `--output-key-file` flag.
+
+If you wish to wait for the CertificateRequest to be signed and store the x509 certificate in a file, you can set
+the `--fetch-certificate` flag. The default timeout is 5 minutes, but can be specified with the `--timeout` flag. The default name of the file storing the x509 certificate
```

Comment:
I would be more specific that the timeout is for the certificate being issued, i think that will be more clear for first time users